### PR TITLE
TPC: adding material type for dedx calibration as parameter

### PR DESCRIPTION
**Pull request by matthias-kleiner**

### CODE REVIEW

**Review by wiechula — DISMISSED** [review]
Looks good to me, thanks @matthias-kleiner !

File: Detectors/TPC/workflow/src/CalibdEdxSpec.cxx [hunk 1]
```diff
@@ -131,7 +133,7 @@ DataProcessorSpec getCalibdEdxSpec()
                                                                 false,                          // GRPECS=true
                                                                 false,                          // GRPLHCIF
                                                                 true,                           // GRPMagField
-                                                                false,                          // askMatLUT
```
Comment (sawenzel) [inline Detectors/TPC/workflow/src/CalibdEdxSpec.cxx:134]: not sure about this one. Shouldn't askMatLUT only be activated if `material-type == 2` ?

Comment (matthias-kleiner) [reply Detectors/TPC/workflow/src/CalibdEdxSpec.cxx:134]: Yes, I think you are right, thanks. I will change this.

**Review by sawenzel — APPROVED** [review]

**Comment by sawenzel** [comment]
Merging (previously approved by @wiechula)

**Pull request merged**